tests/aws/services/sqs/test_sqs.py

```diff
@@ -1085,19 +1084,18 @@ def test_external_hostname_via_host_header(self, monkeypatch, sqs_create_queue):
         queue_name = f"queue-{short_uid()}"
         sqs_create_queue(QueueName=queue_name)
 
-        edge_url = config.get_edge_url()
```

Comment:
⚠️ Are we correct assuming this is an external URL here?
External `localhost.localstack.cloud` vs. internal `localhost`

Reply:
I think so, since it is asserting what the create queue call returns (which is returned to the user), and is in fact the same as the implementation in `sqs/models.py`.